microsoft/PowerToys
# [Video Conference Mute] Push to Talk (Reverse)

## Summary of the Pull Request
Adds push to talk hotkey in settings. Allows to temporarily toggle state of microphone, so you can push to talk or push to mute. When hotkey is released then state return to previous. 


## PR Checklist

- [x] **Closes:** #6308, #20491, #18991
- [x] **Tests:** Not added, all pass
- [x] **Dev docs:** No dev docs for this module
- [x] **New binaries:** - no new binaries
- [x] **Documentation updated:** If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) and link it here: [#4348](https://github.com/MicrosoftDocs/windows-dev-docs/pull/4348)


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
- change hotkey in settings and check if is saved
- push to reverse on muted microphone
- push to reverse on unmuted microphone
- press push to reverse hotkey and release shift or windows only
